### ISSUE TITLE
How to check if my firmware is updating automatically or not?

### ISSUE THREAD

**Issue by xenstar** [DHDAXCW/NanoPi-R2S-2021#18]
Hello bro, thanks for working on a stable release for R2S. This firmware is very stable.

I have downloaded this file and flashed my R2S on 2nd March. Now how do I know if it's automatically updating when you release a new version? Or if I have to manually set a cron job, can you guide me on how to do it?

![image](https://user-images.githubusercontent.com/55502651/110498763-90a4cf00-8132-11eb-82ba-253b2c141738.png)

![image](https://user-images.githubusercontent.com/55502651/110498137-065c6b00-8132-11eb-91e9-f7fc70c0c045.png)

**Comment by xenstar** [comment]
Alright got it, so far it's stable for me. Only some apps, e.g Instagram, What's app call can't consume much bandwidth on phone. if I use the v2rayNG app on android, then the speed is very high. I can show you a video if you want.

**Comment by xenstar** [comment]
I downloaded the file and updated here, but then it just reboots after few time but seems the new update is not flashed. 

![flash](https://user-images.githubusercontent.com/55502651/110498763-90a4cf00-8132-11eb-82ba-253b2c141738.png)

![flash](https://user-images.githubusercontent.com/55502651/110651278-44bb5e00-81f6-11eb-996e-f3ad6e15659e.png)

**Comment by xenstar** [comment]
Can you install R2S Flasher, because before I can easily upgrade using the R2S flasher. 
https://github.com/songchenwen/nanopi-r2s/tree/master/luci-app-r2sflasher

**Comment by DHDAXCW** [comment]
这个好东西，我必须尝试！感谢提供插件

**Comment by xenstar** [comment]
Yeah you can check this one too, it has individual IPK file - https://github.com/kongfl888/luci-app-r2sflasher

Other firmware I was using, with r2s I can easily flash, but your files I can't flash with it. But this is very useful you know. 

Flashing using your SD card, then it takes time to do the all settings.

**Comment by DHDAXCW** [comment]
我今天刚测试了你给的两个链接，分别测试，结果都一样，原因是，我的固件内是lean的源码，不支持此固件升级，但是这升级插件，需要支持FriendlyWrt所有固件才行的。
不知道有何指教，能不能说下详细步骤和过程？

**Comment by xenstar** [comment]
Actually I also don't know about coding much. Before this I was using this one - https://github.com/biliwala/nanopi-openwrt

And I can easily update the new version, with r2s flasher. Since your one is mixture of Lean&Lienol, maybe that's what creating the problem.

Anyways, take time, I am pretty sure you can figure out a solution.

**Comment by DHDAXCW** [comment]
我今天刚通过升级备份里面刷固件，可以有效的升级了最新版。但是你发的链接我看了是friendly的源码，所以你给的插件就支持friendly的源码升级。

**Comment by xenstar** [comment]
You mean you flash the firmware from here right? 

![image](https://user-images.githubusercontent.com/55502651/110917925-74cd4300-8355-11eb-8e1b-05b8431f3cea.png)

**Comment by DHDAXCW** [comment]
yes  今天测试了 可以没问题的

**Comment by xenstar** [comment]
I am using this one. 
![image](https://user-images.githubusercontent.com/55502651/110918573-469c3300-8356-11eb-9745-26ec8f5bc527.png)

after the successful update, it will change the date here right? Or how to know if the update is successful or not? and your setting was all same right?

![image](https://user-images.githubusercontent.com/55502651/110918667-616ea780-8356-11eb-977b-8b0febc15331.png)

**Comment by DHDAXCW** [comment]
我是通过sq升级，ext4是挂载的东西，不知道咋弄，不过友善，1988都是sq格式，可能是通过sq格式升级。正好我今天也通过sq格式升级，不解压直接升级

**Comment by xenstar** [comment]
Great, since this is solved, I will close it. 

There is one issue with Instagram, google play store and some other software, can't consume much bandwidth, but with v2rayNG application, more bandwidth can be acheived.

**Comment by xenstar** [comment]
Now I just updated the sq version. Kernal version and date didn't change. I guess the update is not successful.. 


![autoupdate](https://user-images.githubusercontent.com/55502651/111174300-4a43ea00-85e2-11eb-81b9-61331e006527.png)
